README.md

```diff
@@ -33,22 +33,40 @@ for helm.
 You can `go get` this repo or `git clone` it to start poking around right away.
 
 The project comes ready with a minimal example service that you can easily
-deploy and begin iterating on:
+deploy and begin iterating on.
+
+### Get the project
 
 ```console
 $ go get github.com/pmorie/osb-starter-pack/cmd/servicebroker
-# OR clone the repo
+```
+
+Or clone the repo:
+
+```console
 $ cd $GOPATH/src && mkdir -p github.com/pmorie && cd github.com/pmorie && git clone git://github.com/pmorie/osb-starter-pack
+```
 
-# change into the project directory
+Change into the project directory:
+
+```console
 $ cd $GOPATH/src/github.com/pmorie/osb-starter-pack
+```
+
+### Deploy broker using Helm
 
-# You can deploy two different ways: helm or openshift templates
+Deploy with Helm and pass custom image and tag name.
+Note: This also pushes the generated image with docker.
 
-# deploy with helm
-$ make deploy-helm
+```console
+$ IMAGE=myimage TAG=latest make push deploy-helm
+```
 
-# deploy with openshift templates (Note: You must already be logged into an OpenShift cluster)
+### Deploy broker using Openshift
+
+Note: You must already be logged into an OpenShift cluster.
+
+```console
 $ make deploy-openshift
```

Comment:
Can you add `IMAGE`, `TAG` and `push` here similar to helm?

Reply:
Done.